doc/src/manual/documentation.md

```diff
@@ -81,7 +81,23 @@ As in the example above, we recommend following some simple conventions when wri
    ...
    """
    ```
-5. Include any code examples in an `# Examples` section.
+5. Provide hints to similar functions
+   
+   Sometimes there are functions of similar functionality, which are named quite different. To increase 
```

Comment:
Perhaps `similar functionality` -> `related functionality`?
Also, `which are named quite different` is not needed; it is useful to list related functions even if they have similar names.